### PR TITLE
Copy ara env files only when ara_enable is set true

### PR DESCRIPTION
**Pull request by grawert**

### CODE REVIEW

File: tasks/manager.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     - "{{ manager_docker_compose_directory }}"
     - "{{ manager_configuration_directory }}"
 
-- name: Copy environment files
+- name: Copy environment files for Ara
```
Comment (berendt) [inline tasks/manager.yml:14]: ara or ARA

File: tasks/manager.yml [hunk 2]
```diff
@@ -23,6 +23,7 @@
     - ara-server
     - ara-web
     - database
+  when: ara_enable == true
```
Comment (berendt) [inline tasks/manager.yml]: Use ``ara_enable|bool`` (ansible-lint: [601] Don't compare to literal True/False)

**Review by berendt — CHANGES_REQUESTED** [review]

File: tasks/manager.yml (hunk 2 above)
Comment (grawert) [reply tasks/manager.yml]: Done

**Comment by berendt** [comment]
Replaced by #33

**Pull request closed without merging**